src/constants.ts

```diff
@@ -11,6 +11,9 @@ export const DOTSNYK_FILENAME = '.snyk';
 export const EXCLUDED_NAMES = [GIT_FILENAME, GITIGNORE_FILENAME, DCIGNORE_FILENAME];
 export const CACHE_KEY = '.dccache';
 export const MAX_UPLOAD_ATTEMPTS = 5;
+export const UPLOAD_CONCURRENCY = 5;
+export const MAX_RETRY_ATTEMPTS = 5; // Request retries on 503 errors
+export const REQUEST_RETRY_DELAY = 30 * 1000; // 30 seconds delay between retries
```

Comment:
I've modelled this on the actual response we get from our backend when performing such requests:
```
<= Response: 502 "\n<html><head>\n<meta http-equiv=\"content-type\" content=\"text/html;charset=utf-8\">\n<title>502 Server Error</title>\n</head>\n<body text=#000000 bgcolor=#ffffff>\n<h1>Error: Server Error</h1>\n<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>\n<h2></h2>\n</body></html>\n"
```
So yes, it would take a max of 5 * 30s now. I'm up for suggestions on this ;)